### ISSUE TITLE
Secondary immunodeficiency

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6162]
**Preferred term label:**
Secondary immunodeficiency

**Synonyms**
SID

**Definition (free text, please give PubMed ID)**
Secondary immunodeficiencies result from a variety of factors that can affect a host with an intrinsically normal immune system, including infectious agents, drugs, metabolic diseases, and environmental conditions.; PMID: 20042227 

**Parent term (use hpo.jax.org/app)**
Immunodeficiency

**Comment by pnrobinson** [comment]
This term refers to the cause of immunodeficiency as seen in a group of diseases ("secondary"). The HPO does not encode etiologies, so this is not a good term.
